code/modules/client/preference_setup/global/preferences.dm

```diff
@@ -223,6 +223,14 @@ var/list/_client_preferences_by_type
 	key = "EXAMINE_MESSAGES"
 	options = list(GLOB.PREF_SHOW, GLOB.PREF_HIDE)
 
+/datum/client_preference/fullscreen_mode
+	description = "Toggle Fullscreen Mode"
+	key = "TOGGLE_FULLSCREEN"
+	options = list(GLOB.PREF_NO, GLOB.PREF_YES)
```

Comment:
Any particular reason for switching Yes/No around? It makes it inconsistent with other settings using the same options.
Should instead be able to instead do `default_value = GLOB.PREF_NO`